pages/500.tsx

```diff
@@ -0,0 +1,37 @@
+import Head from "next/head";
+
+import { useHasMounted } from "../app/hooks/useIsMounted";
```

Comment:
Lmao I just realised I named the file wrong 🤦🏼

Reply:
// todo hahah